src/app/tests/suites/certification/Test_TC_VALCC_1_1.yaml

```diff
@@ -0,0 +1,186 @@
+# Copyright (c) 2024 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 62.1.1. [TC-VALCC-1.1] Global attributes with server as DUT
```

Comment:
This should be mostly tested with the Conformance test. Otherwise the test is 100% dependent on PICS being correct.

Reply:
Agreed, we did not yet have agreement to drop the 1.1 test cases in the test plan group, so I have added this test script to be on the safe side. If the agreement is to drop these, it will just be removed in a follow up PR.

As far as I understand this will be discussed as a post-TE2 topic.